bcgov/House-Innovations-HOUSNAV
# HOUSNAV-188: Checkbox Card Component

[HOUSNAV-188](https://hous-bssb.atlassian.net/browse/HOUSNAV-188)

## Overview & Purpose
Create a reusable component for the checkbox card selection on the new walkthrough build wizard page

## Summary of Changes
- Created a Checkbox Card component and added temporary instances to the multi dwelling unit page
- Added additional icon
- Added and refactored cypress tests for building type scenarios
- Refactored some global CSS that didn't need to be global plus renamed a few for better utility purposes

## Testing
- Click on the Multi Unit Dwelling card on the landing page
- See the Checkbox Cards on the Multi Unit Dwelling landing page
- Click to see the selected state

## Checklist
